students/kegan/session03/string_formatting_lab.py

```diff
@@ -0,0 +1,65 @@
+"""
+Kathryn Egan
+
+"""
+
+
+def pad(integer, length):
+    """ Pads given integer as string with 0s up to given length.
```

Comment:
everything in this lab can be done pretty directly with the right formatting specifier..